debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SourceInfoTest.java

```diff
@@ -5,28 +5,40 @@
  */
 package io.debezium.connector.sqlserver;
 
+import static org.fest.assertions.Assertions.assertThat;
+
 import org.junit.Before;
 import org.junit.Test;
 
-import static org.fest.assertions.Assertions.assertThat;
+import io.debezium.relational.RelationalDatabaseSourceInfo;
+import io.debezium.relational.TableId;
 
 public class SourceInfoTest {
 
     private SourceInfo source;
+    private TableId tableId;
 
     @Before
     public void beforeEach() {
         source = new SourceInfo("serverX");
         source.setChangeLsn(Lsn.NULL);
+        tableId = new TableId("c", "s", "t");
     }
 
     @Test
     public void versionIsPresent() {
-        assertThat(source.struct().getString(SourceInfo.DEBEZIUM_VERSION_KEY)).isEqualTo(Module.version());
+        assertThat(source.struct(tableId).getString(SourceInfo.DEBEZIUM_VERSION_KEY)).isEqualTo(Module.version());
```

Comment:
Can you add an assertion to one of the integration tests, too? It seems the s sourced from the catalog id; does this match with the configured database name for SQL Server?